src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/Azure.Configuration.Tests.csproj

```diff
@@ -12,7 +12,22 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Azure.Configuration\Azure.Configuration.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\..\Azure.Base\data-plane\Azure.Base.Tests\Testing\TestPool.cs" Link="TestPool.cs" />
+  </ItemGroup>
+
+  <!-- Project references -->
+  <ItemGroup Condition="'$(UseProjectReferenceToBase)'=='true'">
     <ProjectReference Include="..\..\..\Azure.Base\data-plane\Azure.Base\Azure.Base.csproj" />
-    <ProjectReference Include="..\..\..\Azure.Base\data-plane\Azure.Base.Tests\Azure.Base.Tests.csproj" />
   </ItemGroup>
+
+  <!-- Package references -->
+  <ItemGroup Condition="'$(UseProjectReferenceToBase)'==''">
+    
+    <PackageReference Include="Azure.Base" Version="1.0.0-preview.1" />
+    <PackageReference Include="System.Text.Primitives" Version="0.1.0-preview1-180216-4" />
```

Comment:
Why are you including System.Text.Primitives conditionally?

Reply:
Isn't this a source package? If so why is it needed to be pulled into the test project?

Reply:
I initially had it there because it wasn't been pulled through via closure anymore, but we should be able to always depend on it.

Reply:
Still not sure why this is needed in the test project but including it all the time makes sense if it is needed.

Reply:
This is a corfxlab library, isn't it? I am pretty sure we can remove this dependency. Assign an issue to me and I will do it tomorrow.

Reply:
I just submitted a PR to do it: https://github.com/Azure/azure-sdk-for-net/pull/5292